### ISSUE TITLE
Test using @supports for colour pickers

### ISSUE THREAD

**Issue by DjCrqss** [DjCrqss/Woot-verlay#38]
Colour pickers are not supported in OBS, resulting in the user having to click the current 'accessibility mode' toggle to change the inputs (not very clear). Using @supports (or js equivalent) should be able to automatically change the input field.

**Comment by DjCrqss** [comment]
Not problem of lack of support on obs, obs browser fails to render popups such as <select> and <input[type=color]> and the obs team has mentioned they are not planning to fix it any time soon...
